presto-main/src/test/java/io/prestosql/operator/scalar/TestIpAddressFunction.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.operator.scalar;
+
+import org.testng.annotations.Test;
+
+import static io.prestosql.spi.type.BooleanType.BOOLEAN;
+
+public class TestIpAddressFunction
+        extends AbstractTestFunctions
+{
+    @Test
+    public void testIpAddressContains()
```

Comment:
Can we add some tests for:
* edge cases (e.g., the addresses at the edges of the CIDR range)
* combinations of IPv6 CIDR + IPv4 address and IPv4 CIDR + IPv6 addresses. I suspect there are some legal combinations (especially IPv6 CIDR + IPv4 addresses) that are legal and we're not handling correctly.

Reply:
Added some test cases. By the way, don't we need to care wrong cidr with correct prefix like `x.x.x.x/1`? It will throw IllegalArgumentException at InetAddresses.forString.

```
java.lang.IllegalArgumentException: 'x.x.x.x' is not an IP string literal.
```

Reply:
Yeah, we should throw PrestoException with INVALID_FUNCTION_ARGUMENT in that case

Reply:
Let me update it.